nixos/lib/test-driver/test-driver.py

```diff
@@ -908,47 +1011,223 @@ def unblock(self) -> None:
         """Make the machine reachable."""
         self.send_monitor_command("set_link virtio-net-pci.1 on")
 
+    def release(self) -> None:
```

Comment:
@kraftnix would this have any impact on the slow tear downs that you had discovered and reported in the devos chat?